hack/run-functional-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#Copyright 2021 The hostpath provisioner Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+KUBEVIRT_PROVIDER=k8s-1.25
```

Comment:
Assuming we plan to update this to newer versions in the future.  Will this variable now need to be set in multiple locations?  Would be nice if it could be set only once in the top-level Makefile.

Reply:
I agree, I need to investigate how to do that, not doing something right in the Makefile right now and the variables are not being passed into the scripts. But for this particular case I want this to work, so I can have the half dozen autoupdate PRs merge which are stuck right now.

Reply:
ok by me.  Maybe add a TODO: in the code or something at least so it's not forgotten?

Reply:
I was planning on guiding the PRs through and immediately fixing it. But until this is merged I can't do any of that.